go/client/cmd_team_show_tree.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2017 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/go-framed-msgpack-rpc/rpc"
+	"golang.org/x/net/context"
+)
+
+type CmdTeamShowTree struct {
+	TeamName  keybase1.TeamName
+	SessionID int
+	libkb.Contextified
+}
+
+func (v *CmdTeamShowTree) ParseArgv(ctx *cli.Context) (err error) {
+	v.TeamName, err = ParseOneTeamNameK1(ctx)
+	if err != nil {
+		return err
+	}
+	return err
+}
+
+func (v *CmdTeamShowTree) Run() (err error) {
+	cli, err := GetTeamsClient(v.G())
+	if err != nil {
+		return err
+	}
+
+	protocols := []rpc.Protocol{
+		NewSecretUIProtocol(v.G()),
+	}
+	if err = RegisterProtocolsWithContext(protocols, v.G()); err != nil {
+		return err
+	}
```

Comment:
Nope.  And you aren't sending SecretUI anywhere in the service, so this is essentially ignored.